docs/src/man/LaPalma_example.md

```diff
@@ -1,11 +1,12 @@
 ```@meta
-EditURL = "<unknown>/LaPalma_example.jl"
+EditURL = "../../../tutorials/LaPalma_example.jl"
```

Comment:
Wouldn't you need to specify the GitHub path here? like 
EditURL = "https://github.com/JuliaGeodynamics/GeophysicalModelGenerator.jl/blob/main/tutorials/LaPalma_example.jl"

Reply:
This was auto-generated from Literate, so I'm hoping it'll work